### PR TITLE
Partial core review update

### PR DESCRIPTION
**Pull request by CoMPaTech**
**This breaks device-ids - again**

### CODE REVIEW

File: custom_components/plugwise-beta/sensor.py [hunk 1]
```diff
@@ -275,7 +275,7 @@ def __init__(self, api, coordinator, name, dev_id, sensor, sensor_type):
             self._name = f"Smile {self._name}"
             self._via_id = None
 
-        self._unique_id = f"{dev_id}-{sensor}"
+        self._unique_id = f"{dev_id}-{sensor_type}"
```
Comment (bouwew) [inline custom_components/plugwise-beta/sensor.py:278]: Sensor_type will not work, will lead to duplicate unique_ids. Use sensor instead.
There can be equal sensors but then the devid will be different.

Comment (CoMPaTech) [reply custom_components/plugwise-beta/sensor.py]: c4090b8 reverts it

**Comment by CoMPaTech** [comment]
Lets leave it here for now - it's nothing major and more might come (and it is breaking)

**Comment by bouwew** [comment]
This changes unique_ids, not device names in HA?
So nothing breaks I think?

**Comment by CoMPaTech** [comment]
I guess because of changed ID's you will end up with stale ones (again)

**Comment by CoMPaTech** [comment]
Closing this one - will create an issue that we still have to align the uniq-ids

**Pull request closed without merging**